styles.css

```diff
@@ -0,0 +1,96 @@
+body {
+  width: 310px;
+  height: 718px;
+}
```

Comment:
Great job @osbentech 🤝

- [ ] According to the Figma design provided, there is a big white space at the bottom of your project. Please fix this white space.
- Your design
![your design](https://user-images.githubusercontent.com/63917725/177217202-671ea5f9-6d55-42bf-9058-5912629e58a3.PNG)

- Figma design
![Figma design](https://user-images.githubusercontent.com/63917725/177217222-61cbb7ed-076d-41d2-a916-a6861305d2c4.PNG)